docs/xml/elements/street_segment.rst

```diff
@@ -6,101 +6,104 @@ geography (i.e., segment) is contained within. The start address house number mu
-|                 |                                         |              |            |practice is to put a    |                        |
-|                 |                                         |              |            |very large number such  |                        |
-|                 |                                         |              |            |as 999999 in this       |                        |
-|                 |                                         |              |            |field.                  |                        |
-|                 |                                         |              |            |                        |                        |
-+-----------------+-----------------------------------------+--------------+------------+------------------------+------------------------+
-| UnitNumber      | xs:string                               | Optional     | Repeats    |The apartment/unit      |If the field is not     |
-|                 |                                         |              |            |number for a street     |present or invalid, the |
-|                 |                                         |              |            |segment. If this value  |implementation is       |
-|                 |                                         |              |            |is present then         |required to ignore it.  |
-|                 |                                         |              |            |**StartHouseNumber**    |                        |
-|                 |                                         |              |            |must be equal to        |                        |
-|                 |                                         |              |            |**EndHouseNumber**.     |                        |
-|                 |                                         |              |            |                        |                        |
-|                 |                                         |              |            |                        |                        |
-+-----------------+-----------------------------------------+--------------+------------+------------------------+------------------------+
++----------------------+-----------------------------------------+--------------+------------+------------------------+------------------------+
+| Tag                  | Data Type                               | Required?    | Repeats?   | Description            | Error Handling         |
+|                      |                                         |              |            |                        |                        |
++======================+=========================================+==============+============+========================+========================+
+| NonHouseAddress      |:ref:`NonHouseAddress                    | Optional     | Single     |The common street       |If the element is not   |
```

Comment:
Also if you don't mind changing ``:ref:`NonHouseAddress <non-house-address>` `` to `` `NonHouseAddress`_`` and editing the header in the next section to match (i.e. StreetSegment.NonHouseAddress to NonHouseAddress), that would be great (read: thanks for doing my work for me) ;)